db/db_block_cache_test.cc

```diff
@@ -1991,10 +1982,20 @@ TEST_P(DBBlockCachePinningTest, TwoLevelDB) {
   const int kNumKeysPerFile = kBlockSize * kNumBlocksPerFile / kKeySize;
 
   Options options = CurrentOptions();
-  // `kNoCompression` makes the unit test more portable. But it relies on the
-  // current behavior of persisting/accessing dictionary even when there's no
-  // (de)compression happening, which seems fairly likely to change over time.
-  options.compression = kNoCompression;
+  // Select one of the available libraries that support dictionary compression
+  if (Zlib_Supported()) {
+    options.compression = kZlibCompression;
+  } else if (LZ4_Supported()) {
```

Comment:
Is there a reason this does not use the first of Compressor::GetDictSupported() ?

Reply:
Updated to use Compressor::GetDictSupported() and pick the first available library that supports dictionary compression. The same was done in table_test (IncreaseCacheReservationForCompressDictBuildingBufferOnBuilderAddAndDecreaseOnBuilderFinish)